src/Node.cpp

```diff
@@ -14,7 +15,14 @@ void Node::start() {
   if (functionName == "focus_frame") {
     page()->setCurrentFrameParent(page()->currentFrame()->parentFrame());
   }
-  finish(&result);
+
+  if (result.hasError()) {
+    finish(&result);
+  } else {
+    JsonSerializer serializer;
+    InvocationResult result2 = InvocationResult(serializer.serialize(result.result()));
```

Comment:
Is there a clearer name than `result2`? Maybe `serializedResult` or something like that?